### ISSUE TITLE
Revel and Versioning

### ISSUE THREAD

**Issue by Emyrk** [revel/cmd#139]
Each time revel updates, all my docker builds break as the recommended way to get the revel cmd is by the `go get` command.

Can a dependency file please be created and maintained? Go Deps or Glide, or something?

**Comment by notzippy** [comment]
Did your builds fail due to a dependency change in revel/cmd or revel/revel ?

**Comment by brendensoares** [comment]
@Emyrk can you provide some error messages that you see from your docker containers?

Also, Revel does have vendor support (thanks to @notzippy). Have you checked it out?

**Comment by ptman** [comment]
There is now vendoring with dep in revel 0.20. That should help with this in the future, right?

**Comment by Emyrk** [comment]
Sorry for the delay in my response. I was able to do some manual dependency management to downgrade revel.

I can get the error logs again later tonight/tomorrow, but it had to do with the revel/modules directory I believe. I ended up checking out the v0.19.0 of `revel/modules`, `revel/cmd`, and `revel/revel`

@ptman If there is vendoring now in 0.20 than that would solve my issues yes. I'll take a look at it, thanks
